packages/js/dependency-extraction-webpack-plugin/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@woocommerce/dependency-extraction-webpack-plugin",
-	"version": "2.4.0",
+	"version": "2.4.1",
```

Comment:
No need to bump versions. The package release scripts handle all version bumping along with changelog compilation tasks.

Reply:
Thanks 🙏🏻